README.md

```diff
@@ -148,6 +152,18 @@ Example Payload
 
 The parent (Project Griffon UI) calls navigateTo whenever the URL is updated. The intent here is to provide deep linking capabilities as well as an opportunity for plugins to be good citizens and limit resources when not visible.
 
+##### receiveConnections
+
+Project Griffon UI sends an array of connections. A connection will look like:
+```
+{
+  namespace, // the namespace of the connector
+  context, // object containing parameters needed to run the connector
+  query: { isLoaded, loading, error }, // defines the state of the remote api query
```

Comment:
when I think query I think of the thing being searched or the url. Could use a better key like `status` or `state` or just put `isLoaded`/`loading`/`error` at top level